Dockerfile

```diff
@@ -8,12 +8,12 @@ RUN npm install
 COPY . .
 ARG nodeEnv=production
 ENV NODE_ENV $nodeEnv
-RUN npm run build && if [[ "$nodeEnv" == "production" ]]; then mv node_modules/node-webcrypto-ossl tmp && rm -rf node_modules && mkdir node_modules && mv tmp node_modules/node-webcrypto-ossl && npm install --no-optional; fi
```

Comment:
do we know what is the purpose of this line?

Reply:
not really. but the node-webcrypto is deprecated and as I pumped the node version to 20, it is not required.
So I just remove this line.
To your point if someone knowledgeable on node.js in prod could have a look, there might be a reason to manipulate the node-module directory for prod.